src/box/iproto.cc

```diff
@@ -2899,6 +2904,11 @@ iproto_do_cfg_f(struct cbus_call_msg *m)
 				diag_raise();
 			break;
 		case IPROTO_CFG_STOP:
+			ERROR_INJECT(ERRINJ_IPROTO_CFG_STOP_FAILED, {
+				diag_set(ClientError, ER_INJECTION,
+					 "iproto stop listen");
+				diag_raise();
+			});
```

Comment:
Error injections are supposed to make it easier to get an error which can happen in reality. This branch (`IPROTO_CFG_STOP`) can not fail. There are no possible errors except your injection. Which means the injection does not help to test real behaviour - it just adds new normally unreachable behaviour which you in turn tested.

So this injection should go away. The injection above also is questionable. Could you fail the listen somehow normally?

Reply:
No listen can't fail normally, but i think we need to check this branch of code. For stop i add panic, because i rework cbus_call and now it can't fail in case fiber cancelled or wakeup.